libraries/redcore/database/driver.php

```diff
@@ -248,5 +247,203 @@ public static function deleteInstances()
 		{
 			self::$instances = null;
 		}
+
+		/**
+		 * Splits a string of multiple queries into an array of individual queries.
+		 *
+		 * @param   string  $sql  Input SQL string with which to split into individual queries.
+		 *
+		 * @return  array  The queries from the input string separated into an array.
+		 *
+		 * @since   11.1
+		 */
+		public static function splitSql($sql)
```

Comment:
btw you should add those changed functions to the joomla 2.5 database class as well or it will break on 2.5 installations https://github.com/redCOMPONENT-COM/redCORE/blob/develop/libraries/redcore/database/database/mysqli.php
and
https://github.com/redCOMPONENT-COM/redCORE/blob/develop/libraries/redcore/database/database/mysql.php